HISTORY.md

```diff
@@ -1,3 +1,8 @@
+1.10.0 / 2019-12-17
+===================
```

Comment:
There should be a blank line after the header

Reply:
Thank you, added